src/JsValidatorFactory.php

```diff
@@ -131,7 +131,8 @@ public function formRequest($formRequest, $selector = null)
             $formRequest = $this->createFormRequest($formRequest);
         }
 
-        $rules = method_exists($formRequest, 'rules') ? $formRequest->rules() : [];
+        // see #495
+        $rules = method_exists($formRequest, 'rules') ? $this->app->call([$formRequest, 'rules']) : [];
 
         $validator = $this->getValidatorInstance($rules, $formRequest->messages(), $formRequest->attributes());
```

Comment:
It's probably worth doing the same for `messages()` and `attributes()`? I assume you can do the same thing on those?